phac-nml/irida-next
# Fix: Attachments Illumina file pairing

## What does this PR do and why?
_Describe in detail what your merge request does and why._

When running `bcl2fastq` or `bcl2fastq2` with `--no-lane-splitting` outputted fastq files do not include `_LXXX` so we need to support illumina files without the lane numbering. This PR adds in support for those files.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Upload `test/fixtures/TestSample_S1_R1_001.fastq` and `test/fixtures/TestSample_S1_R2_001.fastq` to a sample
2. Verify that the files are paired together and are labelled as `illumina_pe` for type

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
